servers/list-api/src/background/batchImportHandler.ts

```diff
@@ -0,0 +1,229 @@
+import { EventEmitter } from 'events';
+import * as Sentry from '@sentry/node';
+import config from '../config';
+import { type Unleash } from 'unleash-client';
+import { serverLogger } from '@pocket-tools/ts-logger';
+import { QueueHandler } from './queueHandler';
+import { ImportMapping, ImportMessage, QueueConfig } from './types';
+import { getClient } from '../featureFlags';
+import { SavedItemImportInput } from '../types';
+import gql from 'graphql-tag';
+import { print } from 'graphql';
+
+/**
+ * A slightly silly config to override a value conditionally
+ * with a feature flag
+ */
+class ImportHandlerQueueConfig implements QueueConfig {
```

Comment:
(out of scope for this pr): Def see a use case for making an SQS polling package or updating to use https://github.com/bbc/sqs-consumer across the board in a package or something. This is probably now number 4? of a poller we have?

Reply:
yep, I could see us putting the base class in a package since they all extend from it and just implement the process-specific handling. or using another library yeah.